src/deck.tsx

```diff
@@ -4,9 +4,12 @@ import bind from 'bind-decorator';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { PlayingCard } from './card';
+import { CardValue} from './types';
 
 type DeckProps = {
 }
+var drawnCards: CardValue[] = new Array();
+var remainingCards: CardValue[] = new Array();

```

Comment:
We might want to scope these within the CardDeck? Assuming we might want multiple decks active at some point?

Reply:
I can go either way here, just depends what requirements are on for when we make cards. If they don't need access to the deck then yea we can lower the scope without issue